src/MizarModels/include/MizarModels/SamplingWithFrameTrackReportModel.h

```diff
@@ -79,8 +81,15 @@ class SamplingWithFrameTrackReportModelTmpl : public QAbstractTableModel {
     return kColumnsCount;
   };
   [[nodiscard]] QVariant data(const QModelIndex& index, int role) const override {
-    if (index.model() != this || role != Qt::DisplayRole) return {};
-    return QString::fromStdString(MakeDisplayedString(index));
+    if (index.model() != this) return {};
+    switch (role) {
+      case Qt::DisplayRole:
+        return QString::fromStdString(MakeDisplayedString(index));
+      case kSortRole:
+        return MakeSortValue(index);
```

Comment:
Sorry, I don't get this. The type of `Qt::EditRole` seems to be `QString` https://doc.qt.io/qt-6/qt.html

Even assuming `Qt::EditorRole` can be `double`, the change seems to be using `Qt::EditorRole` instead of `kSortRole`.

Reply:
Hmm. You're right. I'm wondering if that changed or if I'm just wrong.

Reply:
And I don't see how naming smth what is intended for sorting in a way that doesn't suggest sorting, help separate concerns.

Reply:
It changes the contract. You're not promising to deliver something for sorting. You're promising to deliver the raw values. That's entirely different.

Reply:
OK. Currently we return pvalue for significance column and sorting role. Is pvalue "raw" value for the column? I'd rather argue, it should be a bool `pvalue < confidence_level`. But it's a nit. 

So should I just rename kSortingRole to kRawValueRole ?

Reply:
```
So should I just rename kSortingRole to kRawValueRole ?
```

This would also work. (I still think though that it's common practice to use the EditRole for raw data. This example from Qt itself is using `int` as the EditRole type: https://code.qt.io/cgit/qt/qtbase.git/tree/examples/widgets/itemviews/spinboxdelegate/delegate.cpp?h=5.15)

Reply:
Just realized the significance column wouldn't be a raw value - which is also not great. So I'd suggest to leave the sorting role, if we don't find a better option.

Reply:
OK, let's use Qt::EditRole then.